### PR TITLE
Admin user profile page#51

### LINKED ISSUE #51: User Story 21, Admin User Profile Page
```
As an admin user
When I visit a user's profile page ("/admin/users/5")
I see the same information the user would see themselves
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

### PR DESCRIPTION
**Pull request by Bradniedt**
-Adds test for admin user seeing default and merchant profile page as each would if they themselves visited their path.
-Implements features to pass tests
-Added some sad path expectations for not seeing links to ensure @user_role was working correctly.
-All tests passing
-One skipped (user enable/disable)

closes #51

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -6,9 +6,10 @@ def index
   end
 
   def show
-    @merchant = User.find(params[:id])
+    @user = User.find(params[:id])
+    @user_role = @user.role
```
Comment (danbriechle) [inline app/controllers/admin/users_controller.rb]: Very nice work here, I was wondering when we were goin to stop using just merchants 👍

**Comment by danbriechle** [comment]
Looks good, Happy to merge once Anna takes a look.
